### DISCUSSION TITLE
selecting proxies to purchase

### DISCUSSION THREAD

**Discussion by h9o9** [KevinBytesTheDust/CVAmp#309] [General]
Was following the guide. was aiming for $22 package but it doesn't seem to be that price atm. Wondering where it says proxy type it says stattic residential - premium. i cant seem to find premium so wondering it shared viasble with static residential.

lastly at the the bottom is the option high Priority network need for this tool?

**Comment by KevinBytesTheDust** [comment]
I will update the guide soon, I would recommend the "Static Residential - Shared" with high-concurrency as a solid start.

**Reply by h9o9** [reply]
thanks! to understand what and how many i purchase proxy wise. its 1 proxy = 1 view correct?
